### PR TITLE
isp-imx: fix build against updated tinyxml2

### PR DESCRIPTION
**Pull request by MaxKrummenacher**
While at it additionally update to 4.2.2.24.0 used with lf-6.1.55-2.2.0 which then also requires EULA v49.

### CODE REVIEW

File: recipes-bsp/isp-imx/isp-imx_4.2.2.24.0.bb [hunk 1]
```diff
@@ -28,6 +27,7 @@ SYSTEMD_SERVICE:${PN} = "imx8-isp.service"
 
 EXTRA_OECMAKE += " \
     -DSDKTARGETSYSROOT=${STAGING_DIR_HOST} \
+    -DCMAKE_INSTALL_LIBDIR=/usr/lib/ \
```
[the remark this answers is not in the document]
Comment (MaxKrummenacher) [reply recipes-bsp/isp-imx/isp-imx_4.2.2.24.0.bb]: Doing a `git status` before pushing would help. ;-)
Done.

File: recipes-bsp/isp-imx/isp-imx_4.2.2.24.0.bb [hunk 2]
```diff
@@ -47,6 +47,11 @@ EXTRA_OECMAKE += " \
     -Wno-dev \
 "
 
+do_configure:prepend () {
```
Comment (otavio) [inline recipes-bsp/isp-imx/isp-imx_4.2.2.24.0.bb:50]:
```suggestion
do_configure:prepend () {
    # FIXME: Should be rebuild.
```

Comment (MaxKrummenacher) [reply recipes-bsp/isp-imx/isp-imx_4.2.2.24.0.bb:50]: Done

**Review by otavio — APPROVED** [review]

**Review by thochstein — APPROVED** [review]
LGTM

**Pull request merged**